### ISSUE TITLE
XAML AutomationId NOT working in UI Tests

### ISSUE THREAD

**Issue by pfafft33** [xamarin/Xamarin.Forms#10527]
### Steps to Reproduce

1. 
2. 
3. 

### Basic Information

- Version with issue: 16.5.4 (Community Edition)
- Last known good version: None
- IDE: Visual Studio Community Edition
- Platform Target Frameworks: .NET Standard 2.0.3
  - iOS:  NA
  - Android: Android 9.0 (Pie)
  - UWP:  NA
- Android Support Library Version: ???
- Nuget Packages:
- Affected Devices:

### Screenshots

![GraphPage_XAML_20200430](https://user-images.githubusercontent.com/47510341/80757879-7eadd500-8afa-11ea-9af2-2dbd73958bd1.PNG)
![GraphPage_Repl_20200430](https://user-images.githubusercontent.com/47510341/80757912-88373d00-8afa-11ea-809e-16fcfd1fdc2a.PNG)

**Comment by pfafft33** [comment]
Yes, I have opened a new copy of a previous issue because somebody CLOSED my previous issue!!!

**Comment by hartez** [comment]
> Yes, I have opened a new copy of a previous issue because somebody CLOSED my previous issue!!!

I can't seem to find any previous issues posted by you. Do you know what the issue number was?

**Comment by hartez** [comment]
@pfafft33 Which version of Xamarin.Forms are you using? I just tested this with 4.4 and 4.6 and the AutomationIds are showing up in the tree in my test project.

**Comment by hartez** [comment]
> Yes, I have opened a new copy of a previous issue because somebody CLOSED my previous issue!!!

Ah, I see what happened. My apologies, you had updated the VS Feedback issue with 

> I got this fixed!
> All I had to do was upgrade from Xamarin.Forms version 4.5.0.530 to version 4.5.0.657 on all the projects in my solution.

so I closed the issue.

**Comment by hartez** [comment]
So are you still on version 4.5.0.657? What happens if you update to 4.6?

**Comment by pfafft33** [comment]
After updating to 4.6.0.726, the two AutomationId's that I was having problems with before now show up.
BUT, if I add a new AutomationId to a different button, it does NOT show up in the Repl, even after doing a Save and Build!!!
I think that you actually have to shut down VS and re-start it, in order to get the newly added AutomationId's to show up in the Repl.

**Comment by pfafft33** [comment]
I did some further testing.
Not EVEN shutting down VS and re-starting gets the AutomationId's to show up in the Repl.
I tried Save / Build
I tried Unload / Reload project
I tried exit VS and re-start
I tried running ALL of my tests and then re-checking the Repl
I tried Clean / Re-build
Nothing I try seems to work. But, SOMEHOW the 2 buttons that I reported earlier are now showing up with their AutomationId's in the Repl.
I don't know WHAT causes the Repl to get updated with new AutomationId's, but it needs to happen much "earlier" in the process.
